portfolio/src/main/webapp/CommentHandler.js

```diff
@@ -0,0 +1,23 @@
+export default class CommentHandler {
+  /**
+   * Loads all comments from backend.
+   * This function is called when the page is loaded.
+   * @returns {JSON Array} a JSON array containing all comments
+   */
+  static async loadComments() {
```

Comment:
See other comment about making this an instance method rather than a static method. Of all the methods I've seen, probably about 0.1% are static methods.